app/src/main/java/com/example/myfinanses/ui/account/AccountActivity.kt

```diff
@@ -2,6 +2,9 @@ package com.example.myfinanses.ui.account
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.example.myfinanses.R
 import com.example.myfinanses.databinding.ActivityAccountBinding
 
 class AccountActivity : AppCompatActivity() {
```

Comment:
class AccountActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAccountBinding

    override fun onStart() {
        super.onStart()

        if (isCurrentUser()) {
            startMainActivity()
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAccountBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val navHostFragment = supportFragmentManager.findFragmentById(R.id.nav_host_fragment) as NavHostFragment
        binding.bottomNavViewAccount.setupWithNavController(navHostFragment.navController)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    private fun isCurrentUser() = authFB.currentUser != null

    private fun startMainActivity() {
        val intent = Intent(applicationContext, MainActivity::class.java).apply {
            flags = (Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK)
        }
        startActivity(intent)
    }
}

Reply:
Sprawdzenie czy user nie jest juz zalogowany, jesli tak to odrazu otwieramy MainActivity